src/libs/EmojiTrie.ts

```diff
@@ -1,29 +1,46 @@
-import _ from 'underscore';
+import React from 'react';
+import {SvgProps} from 'react-native-svg';
 import emojis, {localeEmojis} from '../../assets/emojis';
 import Trie from './Trie';
 import Timing from './actions/Timing';
 import CONST from '../CONST';
+import {MetaData} from './Trie/TrieNode';
+
+type Emoji = {
+    code: string;
+    header?: boolean;
+    icon?: React.FC<SvgProps>;
+    name?: string;
+    types?: string[];
+};
+
+type LangEmoji = {
+    name?: string;
+    keywords: string[];
+};
+
+type LangEmojis = Record<string, LangEmoji>;
 
 Timing.start(CONST.TIMING.TRIE_INITIALIZATION);
 
 const supportedLanguages = [CONST.LOCALES.DEFAULT, CONST.LOCALES.ES];
```

Comment:
```suggestion
const supportedLanguages = [CONST.LOCALES.DEFAULT, CONST.LOCALES.ES] as const;
type SupportedLanguages = (typeof supportedLanguages)[number];
```

We can use `SupportedLanguages` in `createTrie()` function.